context-free.md

```diff
@@ -1,22 +1,33 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to create primarily math/geometry-based images in a
+ (relatively) simple way. 
 
```

Comment:
I thought it was interesting that you specified math/geometry-based images in your description. I hadn't thought of it like that ( I believe I only said images ) but you have a great point that it is very recursive/math based.